### PR TITLE
Ir fixes

### PR DESCRIPTION
**Pull request by lexelby**
A couple little fixes and tests for bugs that appeared while demoing to the rest of the SRE team (of course).  Also adds logging to `rake sync:users`.

@jmervine @reidmix @joshuatobin

### CODE REVIEW

File: app/support/mediators/user/sync.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 module Mediators::User
   class Sync < Mediators::Base
     def call
+      Rails.logger.info("Syncing users from #{Config.chatops_users_url}")
       users.each do |chatops_user|
+        Rails.logger.info(chatops_user)
```
Comment (reidmix) [inline app/support/mediators/user/sync.rb]: Should probably start using the Loggerator `log` commands.

Comment (joshuatobin) [inline app/support/mediators/user/sync.rb]: Oh good catch! +1

Comment (lexelby) [inline app/support/mediators/user/sync.rb]: [Loggerator](https://github.com/heroku/loggerator) has no usage info -- I'd have to dig in the code to know how to use it.  Do we have any plans to fix this?

Comment (reidmix) [inline app/support/mediators/user/sync.rb]: Yes we have plans to fix this.  Once I have a bright spot in all my copious time :P :D <3 Until then:
In the comments of #21 I put a basic cheat sheet.

**Comment by lexelby** [comment]
Oops, totally forgot about this one.  Gonna start fresh.

**Pull request closed without merging**